### PR TITLE
Removed FoodTruck ChangeMenu use case and fixed some syntax erros in Addon and Singleton Controllers

### PR DESCRIPTION
**Pull request by akashilangovan**

### CODE REVIEW

**Comment by akashilangovan** [comment]
Also added FoodTruck Controller

File: src/UseCases/FoodTruck/CreateFoodTruckInputBoundary.java [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+import Entities.Interfaces.IAddon;
 import Entities.Interfaces.IOrderbook;
 import Entities.Menu;
 
+import java.util.HashMap;
+
 public interface CreateFoodTruckInputBoundary {
 
-    Boolean createFoodTruck(String userToken, String name, Menu menu, String status, IOrderbook orderbook, String location);
+    Boolean createFoodTruck(String userToken, String name, Menu menu, String status, HashMap<IAddon, Boolean> addAvail,
```
Comment (stephlum) [inline src/UseCases/FoodTruck/CreateFoodTruckInputBoundary.java:11]: We should refactor the food truck use cases to be shop use cases

**Review by stephlum — COMMENTED** [review]
For the most part it is good, please see comment about food truck use cases

File: src/UseCases/FoodTruck/CreateFoodTruckInputBoundary.java (hunk 1 above)
Comment (akashilangovan) [reply src/UseCases/FoodTruck/CreateFoodTruckInputBoundary.java:11]: Noted

**Pull request merged**